.devcontainer/devcontainer.json

```diff
@@ -8,16 +8,15 @@
     // Append -bullseye or -buster to pin to an OS version.
     // Use -bullseye variants on local arm64/Apple Silicon.
     "args": {
-      "VARIANT": "18-bullseye"
+      "VARIANT": "20-bullseye"
     }
   },
   "customizations": {
     // Add the ids of extensions you want installed when the container is created.
     "extensions": [
       "dbaeumer.vscode-eslint",
       "esbenp.prettier-vscode",
-      "firsttris.vscode-jest-runner",
-      "amazonwebservices.aws-toolkit-vscode"
```

Comment:
Is this intentional and we no longer need the plugin?

Reply:
Yes, it's intentional. 

I've remove it to align this config with the Gitpod one. There I also removed the extension a while ago because it now requires a login due to CodeWhisperer.

Ultimately I think we should just include the bare minimum of extensions needed and this is not one of them.

In the future I'd like to go even further and move away completely from pre installing any extension and instead use the [recommended extension system](https://code.visualstudio.com/docs/editor/extension-marketplace#_recommended-extensions) which would simply prompt the user and let them choose which one they want to install. This is for a future PR though.